docs/docs/migrate-from-v1/README.md

```diff
@@ -0,0 +1,358 @@
+Please note that any values stored in `$.service.db` are only accessible in subsequent workflow runs _in the same step_.
+:::
+
+### Public workflows
+
+At this time, all v2 workflows are private. Unforunately at this time there is no workaround. We'll announce when a workaround for this limiation is available.
+
+If you're working with Pipedream support to troubleshoot your workflow, you can share it with the support team under your workflow's **Settings**.
+
+### Rolling back a specific version
+
+In v2, you can test and save your progress on a workflow _without_ deploying it.
+
+However, after deploying it's not possible to rollback to a prior version of a deployed workflow.
+
+You can still edit a deployed workflow, just like in v1 but automatic version rollbacks are not currently possible.
+
+## FAQs
+
+### What are the benefits of the new (v2) workflow builder?
+
+* **Edit & test** your workflows in separate editing mode without impacting live workflows
+* **Support for multiple languages** including Node, Python, Golang & bash
+* **Granular testing** including the ability to test individual steps and more
+* **Multiple triggers** are now supported per workflow
+* **Improved** forms for easier configuration and streamlined building
+
+### What are the limitations of the new (v2) workflow builder?
+
+* `$checkpoint` has been removed from v2 workflows, but `$.database` is a near equivalent.
+* Sharing workflows is not supported
+* Making workflows public is not supported
+* Workflows are no longer versioned
+
+[The limitations of the new workflow builder are covered in detail here](/docs/migrate-from-v1#limitations).
+
+### Are v2 workflows backwards compatible?
+
+No, v2 workflows are not currently compatible with the v1 builder.
+
+However, pre-built component actions are still compatible across both versions.
+
+### Is the Component API changing as well? Will I need to rewrite Components?
+
+No. Any components in the public registry or any private components you have published in your account are compatible with v2.
+
+The v2 workflow builder utilizes the same Component API allowing you to create components from within your workflows, which was not possible in v1.
+
+### Will I still be able to open and edit v1 workflows?
+
+Yes, absolutely you will still be able to view and edit v1 workflows. There is no need to immediately change your workflows from v1 to v2.
+
+### How do I migrate v1 workflows to v2 workflows?
+
+At this time we do not have an automated process to change v1 to v2. To create a v2 equivalent workflow, you can recompose your v1 workflow in the v2 builder.
+
+However, if it uses custom Node.js code steps, be sure to [follow the changes we describe in the guide above](/migrate-from-v1/#node-js-code-step-changes).
+
+### When will the new (v2) workflow builder be the default builder for all customers?
+
+v2 is currently an opt in feature, you can still create and edit v1 workflows if you have an grandfathered account.
```

Comment:
I would change this to section to read:

By default, existing users will still default to the v1 builder. You can create new v2 workflows from the dropdown menu next to the **New** workflow button.

if you'd like to default to the v2 builder when creating new workflows, you can change the **Builder Version** in [your account settings](https://pipedream.com/settings/account).

Reply:
Right, forgot we added that setting good call out